src/libs/ajax.js

```diff
@@ -96,13 +96,20 @@ export const Buckets = {
   notebook: (namespace, bucket, name) => {
     const bucketUrl = `storage/v1/b/${bucket}/o`
 
+    const copy = async newName => {
+      return fetchBuckets(
+        `${bucketUrl}/${nbName(name)}/copyTo/b/${bucket}/o/${nbName(newName)}`,
+        _.merge(authOpts(await Sam.token(namespace)), { method: 'POST' })
+      )
+    }
+    const doDelete = async () => {
```

Comment:
Is this just to avoid a name collision? If so, I might suggest `delete_` instead of adding the (arguably worthless) verb "do."